lib/course/widgets/course_detail_screen.dart

```diff
@@ -36,16 +36,17 @@ class CourseDetailsScreen extends StatelessWidget {
           omitHorizontalPadding: true,
           sliver: SliverList(
             delegate: SliverChildListDelegate([
-              Padding(
-                padding: EdgeInsets.symmetric(
-                  vertical: 18,
-                  horizontal: 12,
+              if (course.description != null) ...[
```

Comment:
As highlighted by @janrenz, what if the course description is empty? Then we shouldn't display the text either.
```suggestion
              if (course.description != null || course.description.isEmpty) ...[
```

Reply:
Sorry, I was expecting the server to normalize this to `null`…

Reply:
Hmm come to think of it, we can probably even use [`.isBlank`](https://pub.dev/packages/dartx#isblank) from the <kbd>dartx</kbd> package:
```suggestion
              if (course.description != null && !course.description.isBlank) ...[
```

Reply:
Implemented, though that's definitely a responsibility of the server

Reply:
Yes, I agree this can be defined as an API requirement.